docs/src/openapi-v1.yaml

```
@@ -1451,10 +1452,16 @@ components:
           type: boolean
           description: Whether the given address is valid ss58 formatted.
         ss58Prefix:
-          type: string
+          type: number
```

```suggestion
          type: string
```

So this is actually a string, since the sanitization process will serialize it as a string. 
That being said, you can add a `format` field beneath the description with `unsignedInteger` to denote it will be a number.